### DISCUSSION TITLE
Possible to copy a row or cell item?

### DISCUSSION THREAD

**Discussion by mhkeller** [perspective-dev/perspective#2356] [Q&A] [answered]
The "copy all" button is very useful when you want all of the data but is there a way to copy a single row (or multiple by highlighting via command-click or shift-click)? Similarly, is it possible to copy just a single cell value? 

The [editable](https://perspective.finos.org/block/?example=editable) example seems to have the proper UX for clicking into a cell and selecting its contents except I don't want the data to be editable.

Thanks for the help.

edit: I see there's a `selectable` option but that seems to only highlight and not affect the copy functionality.

**Comment by lmeyerov** [comment]
Any more thoughts here? This is the more typical case for us than download-all: selectable for copy, no edit

**Reply by texodus** [reply, marked as answer]
#2618 adds row, column and region selection modes to copy/export.
